Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Feedback.hs

```diff
@@ -36,14 +36,16 @@ import Tools.Error
 data FeedbackReq = FeedbackReq
   { rideId :: Id DRide.Ride,
     rating :: Int,
-    feedbackDetails :: Maybe Text
+    feedbackDetails :: Maybe Text,
+    feedbackChips :: Maybe [String]
   }
   deriving (Show, Generic, ToJSON, FromJSON, ToSchema)
 
 data FeedbackRes = FeedbackRes
   { bppBookingId :: Id DBooking.BPPBooking,
     ratingValue :: Int,
     feedbackDetails :: Maybe Text,
+    feedbackChips :: Maybe [String],
```

Comment:
Is it correct implementation? In task description said, that api should use some predefined chips, not arbitrary `Text`:

> We need to develop a backend API that allows customers to provide feedback for riders. The API should support the inclusion of chips, which are predefined tags or categories, to facilitate feedback classification.
> 
> The API should enable customers to submit their feedback along with one or multiple chips selected from a predetermined list. This will help us organize and analyze feedback more effectively, gaining insights into specific areas of improvement.
> 
> Additionally, the API should handle data validation, ensuring that the feedback and chip selection are within the specified limits and formats.

Reply:
created the predefined chips.